snakemake/__init__.py

```diff
@@ -334,6 +336,9 @@ def snakemake(
         export_cwl (str):           Compile workflow to CWL and save to given file
         log_handler (function):     redirect snakemake output to this custom log handler, a function that takes a log message dictionary (see below) as its only argument (default None). The log message dictionary for the log handler has to following entries:
         keep_incomplete (bool):     keep incomplete output files of failed jobs
+        nobenchmark (bool):         Disable benchmarking for all rules
+        benchmark_output (str):            Write all benchmared stats to given file (tab-separated)
+        benchmark_all (str):        Benchmark all rules regardless if benchmark directive is set in Snakefile. Write all benchmark stats to given file (tab-separated)
```

Comment:
It might be neat to automatically generate the benchmark filename if none is provided, somewhere in `.snakemake` with a datestamp of when the run was started.

That way one wouldn't need to change the name when rerunning and not wanting to overwrite an existing benchmark output.

Reply:
Added